rules/valid-expect.js

```diff
@@ -5,8 +5,26 @@
  * MIT license, Tom Vincent.
  */
 
+const util = require('./util');
+
 const expectProperties = ['not', 'resolves', 'rejects'];
 
+function isResolvesOrRejectsExpression(node) {
+  if (node.expression.type !== 'CallExpression') {
+    return false;
+  }
+  const callee = node.expression.callee;
+  return (
```

Comment:
This could be improved with something like [`_.get`](https://lodash.com/docs/4.17.5#get) or [`get-value`](https://www.npmjs.com/package/get-value).

Reply:
I'm down with that. Will make coverage a bit easier as well 🙂

Reply:
Do you have preferences over which dependency to add? `get-value` is smaller, although I don't know if that's as big a deal with ESLint plugins (for example, I would be more concerned about adding the entire Lodash dependency for a web app than a command line tool).

Reply:
no preference 🙂